test/parallel/test-http2-serve-file.js

```diff
@@ -1,38 +1,32 @@
 'use strict';
 
 const common = require('../common');
+const { readSync, readKey, path } = require('../common/fixtures');
```

Comment:
In this case I think it would be best to not use destructuring because `path` is a Node.js module and the name is conflicting. It is just confusing that it does not stand for the same.

Reply:
Thank you for the feedback. I addressed the issue.